service/service.go

```diff
@@ -146,19 +166,98 @@ func New(config Config) (*Service, error) {
 		}
 	}
 
-	var createService *create.Service
+	var legacyResource framework.Resource
+	{
+		legacyConfig := legacyresource.DefaultConfig()
+		legacyConfig.AwsConfig = awsConfig
+		legacyConfig.AwsHostConfig = awsHostConfig
+		legacyConfig.CertWatcher = certWatcher
+		legacyConfig.CloudConfig = ccService
+		legacyConfig.K8sClient = k8sClient
+		legacyConfig.KeyWatcher = keyWatcher
+		legacyConfig.Logger = config.Logger
+		legacyConfig.PubKeyFile = config.Viper.GetString(config.Flag.Service.AWS.PubKeyFile)
+
+		legacyResource, err = legacyresource.New(legacyConfig)
+		if err != nil {
+			return nil, microerror.Mask(err)
+		}
+	}
+
+	var cloudformationResource framework.Resource
+	{
+		cloudformationConfig := cloudformationresource.DefaultConfig()
+
+		cloudformationConfig.CertWatcher = certWatcher
+		//cloudformationConfig.CloudConfig = ccService
```

Comment:
What happens with that?

Reply:
This was also from the migration branch. The cloudconfig service is now passed in for the stub.